client/src/views/routes.js

```diff
@@ -109,6 +110,13 @@ const routes = [
     exact: true,
     login: false,
     component: PartnerOrganisationList
+  },
+
+  {
+    path: '/you-are-banned-what-now/:uid',
+    exact: false,
+    login: false,
+    component: YouAreBannedWhatNow
```

Comment:
We should add this route again below without `:uid` and `exact: true`. Then we do not need to link to `111...` for it to work,